test/camera/camera_test.py

```diff
@@ -0,0 +1,391 @@
+from os import path
+
+import sys
+import unittest
+
+FILE_PATH = path.dirname(path.realpath(__file__))
+sys.path.append(FILE_PATH + "/../../src")
+
+from camera import Camera
+
+class CameraTest(unittest.TestCase):
+    def setUp(self):
+        self._db = filedb.fileDB(FILE_PATH + "/../camera/config")
+        self._db.set('pan_offset', 0)
```

Comment:
Why not just make these 0 in the config?

Reply:
I need them to be set for the relative calibration tests, if the value is not 0 I can't reliably pinpoint the bound and 1 beyond the bound.

Reply:
But you're writing 0 to it anyway. Why not just make it 0 in the config file itself?